specification/containerservice/resource-manager/readme.md

```diff
@@ -34,10 +34,20 @@ These are the global settings for the ContainerServices API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2019-02
+tag: package-2019-04
 ```
 
+### Tag: package-2019-04
 
+These settings apply only when `--tag=package-2019-04` is specified on the command line.
```

Comment:
You also need to add section for : 

```
### Tag: package-2019-04-only

These settings apply only when `--tag=package-2019-04-only` is specified on the command line.

``` yaml $(tag) == 'package-2019-04-only'
input-file:
- Microsoft.ContainerService/stable/2019-04-30/openShiftManagedClusters.json
```

around line 127

Reply:
Thanks added.